### PR TITLE
migrate ci load test runs off of hub

### PR DESCRIPTION
**Pull request by molinamelendezj**
As hub sunsets this PR achieves:

1. temporary run nightly load tests on circle ci xlarge 8cpus agents (load test require 7) on minikube
2. temporary run ci load tests on circle ci xlarge 8cpus agents (load test require 7) on minikube
3. upload PPS and PFS load test results stats to bigquery to start monitoring performance a bit closer. 

sample stats
Screen Shot 2022-04-21 at 4 04 42 PM

upcoming work will remove dependency of minikube load test run to achieve scaled load tests. this temporary for hub shut down.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -127,31 +124,50 @@ jobs:
...
       - run: cr upload -o pachyderm -r helmchart --skip-existing
       - run: cd helmchart && cr index -o pachyderm -r helmchart -c https://helm.pachyderm.com --package-path ../.cr-release-packages --push
-  hub:
-    resource_class: large
+  load_test:
+    resource_class: xlarge
     machine:
       image: ubuntu-2004:202101-01
+    environment:
+      GOOGLE_PROJECT_ID: build-release-001
```
Comment (jrockway) [inline .circleci/config.yml:132]: Does `bq` just automatically pick these variables up?

**Review by jrockway — APPROVED** [review]
Extremely good work!  Thanks for doing this.

**Comment by jrockway** [comment]
One more thing though, can we rename `run_hub_tests.sh` to like `run_load_tests.sh`?

File: .circleci/config.yml (hunk 1 above)
Comment (molinamelendezj) [reply .circleci/config.yml:132]: yea, the circle ci gcp cli orb does when it sets up the cli on agent

Comment (jrockway) [reply .circleci/config.yml:132]: got it, so that orb is doing like `gcloud init` or whatever.

**Pull request merged**